avocado-core/src/main/scala/org/bdgenomics/avocado/genotyping/BiallelicGenotyper.scala

```diff
@@ -313,13 +388,16 @@ private[avocado] object BiallelicGenotyper extends Serializable with Logging {
    * @param rdd An RDD containing the product of joining variants against reads
    *   and then grouping by the reads.
    * @param ploidy The assumed copy number at each site.
+   * @param scoreAllSites If true, calculates likelihoods for all sites, and not
+   *   just variant sites.
    * @return Returns an RDD of (Variant, Observation) pairs.
    */
   private[genotyping] def readsToObservations(
     rdd: RDD[(AlignmentRecord, Iterable[Variant])],
-    ploidy: Int): RDD[(Variant, Observation)] = ObserveReads.time {
+    ploidy: Int,
+    scoreAllSites: Boolean): RDD[(Variant, Observation)] = ObserveReads.time {
```

Comment:
`scoreAllSites` gets passed around a few times, is it worth creating a scoring strategy or abstract class + subclasses for `BiallelicGenotyper`?

Reply:
I'd rather not.